### DISCUSSION TITLE
Running renovate in Tekton (Openshift pipeline)

### DISCUSSION THREAD

**Discussion by gnaponie** [renovatebot/renovate#26640] [Suggest an Idea]
### Tell us more.

Hello,
I'm new to the project, so I hope I'm doing everything right :)
We are going to try to run renovate on Tekton (specifically Openshift pipeline).
Our current environment has several renovate runners configured in github and gitlab. The idea is to unify these instances into one running with tekton.
This seems a good solution because it will allow us to have a unique place, stable, reliable and also that automatically comes with a nice view and easily configurable.
I just wanted to get your thoughts on that, before proceeding with my idea. Do you have any insight on that? Thank you

Cheers

**Comment by rarkins** [comment]
Renovate ideally can run almost anywhere when it's container-based, so it sounds like a fine idea. The images are intended to run on OpenShift (group 0 etc)
